articles/libraries/when-to-use-lock.md

```diff
@@ -1,122 +1,79 @@
-        </tr>
-        <tr>
-            <td>Remembers the last used Connection for a given user.</td>
-            <td>X</td>
-            <td></td>
-        </tr>
-        <tr>
-            <td>Automatically accommodates internationalization.</td>
-            <td>X</td>
-            <td></td>
-        </tr>
-        <tr>
-            <td>Automatically provides password policy checking during sign up.</td>
-            <td>X</td>
-            <td></td>
-        </tr>
-        <tr>
-            <td>Follows strict requirements set by your company in terms of appearance.</td>
-            <td></td>
-            <td>X</td>
-        </tr>
-        <tr>
-            <td>Allows for expert usage of HTML, CSS, and JavaScript.</td>
-            <td></td>
-            <td>X</td>
-        </tr>
-        <tr>
-            <td>Adapts to a simpler business process of just a few use cases (e.g. username/password and social provider authentication).</td>
-            <td></td>
-            <td>X</td>
-        </tr>
-        <tr>
-            <td>Handles multiple database or Active Directory Connections.</td>
-            <td></td>
-            <td>X</td>
-        </tr>
-    </tbody>
-</table>
+---
+description: When adding Auth0 to your apps, you have the option to implement either the Lock login widget, or a custom UI. This page will help you choose.
+---
+
+# Lock vs. a Custom UI
+
+When adding Auth0 to your apps, you have the option to implement either:
+
+* [Lock](/libraries/lock)
+* A custom user interface that you have created in conjunction with the [auth0.js](libraries/auth0js) library or the [Authentication API](/auth-api)
+
+**Lock** is a drop-in authentication widget that provides a standard set of behaviors and a customizable user interface. **Auth0.js** is a client-side library that *does not* come with a user interface but allows for expanded customization of the behavior and appearance of the login. The **Authentication API** provides integration without requiring the use of Auth0 libraries. The best option to choose will depend on the needs of your app.
+
+### When to Implement Lock vs. a Custom UI
+
+|Desired UI Attributes:|Lock|Custom UI|
+|:---|:---:|:---:|
+|Possesses the structure, look, and feel of Lock.|X||
+|Has a simple design that fits in with most modern websites with just a few tweaks to its custom color.|X||
+|Adapts to your configuration and only show the allowable options in the appropriate situations.|X||
+|Chooses the correct connection automatically.|X||
+|Remembers the last used connection for a given user.|X||
+|Automatically accommodates internationalization.|X||
+|Automatically provides a password policy checking at sign up.|X||
+|Follows strict appearance requirements as set by your company.||X|
+|Allows for expert usage of HTML, CSS, and JavaScript.||X|
+|Adapts to a simpler process for username/password and social provider authentication.||X|
+|Handles multiple database or Active Directory Connections.||X|
 
```

Comment:
@Catografix Can we change the table ot be like auth0.com/docs/support. The Yes/No should be green or red.